packages/vertica-nodejs/README.md

```diff
@@ -222,6 +222,29 @@ Connection strings work the same way with connection pools
     })
 ```
 
+## TLS
+
+### TLS Modes
+
+Current TLS Support in vertica-nodejs is limited to server modes that does not require the client to present a certificate. mTLS will be supported in a future version of vertica-nodejs. 
+
+Valid values for the tls_mode connection property are `disable`, `require` which will ensure the connection is encrypted, `verify-ca` which ensures the connection is encrypted and the client trusts the server certificate, and `verify-full` which ensures the connection is encrypted, the client trusts the server certificate, and the server hostname has been verified to match the provided server certificate. 
```

Comment:
nit: Make `tls_mode` in this sentence a code snippet to be consistent.

Reply:
You and your darn attention to detail..

Fixed